### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by arvaibhav**
- refactored sync_audio_files from update database
- separate out dao ( Data Access Object design pattern)  from views 
- separate out utils from views 
- removed direct db models dependency from views

### CODE REVIEW

File: waveform_audio/dao.py
Comment (unna97) [inline waveform_audio/dao.py:1]: Shouldn't we convert the SET &  GETs to apis?

**Review by unna97 — CHANGES_REQUESTED** [review]
Can we change the functions to apis? Also, maybe we also need to rename functions, currently it's not very clear

File: waveform_audio/dao.py
Comment (arvaibhav) [reply waveform_audio/dao.py:1]: Only if apis required on client side , unnecessary not required to build with test cases. However dao function still be used in same way as api https controller should not directly access db as per standard design patterns.

Comment (unna97) [reply waveform_audio/dao.py:1]: I will requires apis on client side eventuatlly.

**Pull request closed without merging**